runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8326,7 +8338,10 @@ done:;
 		J9Class * volatile resolvedClass = ramCPEntry->value;
 
 		if ((NULL != resolvedClass) && J9_IS_J9CLASS_VALUETYPE(resolvedClass) && !VM_VMHelpers::classRequiresInitialization(_currentThread, resolvedClass)) {
-			j9object_t instance = _objectAllocate.inlineAllocateObject(_currentThread, resolvedClass);
+			j9object_t instance = NULL;
+			if (J9_ARE_NO_BITS_SET(resolvedClass->classFlags, J9ClassContainsUnflattenedFlattenables)) {
```

Comment:
Code should be ifdeffed as above.

Reply:
@gacholio It looks like the whole block of `defaultValue` Bytecode has already been ifdef'd.

Reply:
Fair enough.